lib/dotenv-flow.js

```diff
@@ -74,12 +74,14 @@ function config(options = {}) {
     process.env.NODE_ENV = options.default_node_env;
   }
 
+  const node_env = options.node_env || process.env.NODE_ENV
```

Comment:
Here it is, this one place. It is correct, but don't forget to set `;` at the end of the line.

Reply:
I have the styling issue fixed. However, doesn't the line above cover your first case or am I missing something?

Reply:
Treat this comment as an extension of this: https://github.com/kerimdzhanov/dotenv-flow/pull/6/files#r241584549

Reply:
oh ok. I've removed the other references

Reply:
For this particular case I know you wanted to remove all other references of `process.env.NODE_ENV` 
However in the case that `options.node_env` is not included but `options.default_node_env` is included. Wouldn't we still want `process.env.NODE_ENV` to be set to the default provided by the user like I currently have below? or would we like to change that so setting `process.env.NODE_ENV` is up to the user either inside .env files or their own code?

```
  if (!process.env.NODE_ENV && options.default_node_env) {
    process.env.NODE_ENV = options.default_node_env;
  }

  const node_env = options.node_env || process.env.NODE_ENV;
```

Reply:
I think it should be something like:

```
cont node_env = options.node_env || process.env.NODE_ENV || options.default_node_env;
```

One liner logic, where we see the exact priority from right to left ;)

Reply:
Actually I gave this some thought. Is this more along the lines of what we are after 
```
const node_env = options.node_env || options.default_node_env || process.env.NODE_ENV || undefined;
```

That way the each value is evaluated in that order and defaults to undefined if nothing else is available

Reply:
And use this variable in all other places where we need to refer current environment name.

Reply:
haha ok. I should have refreshed my page. I see we ended up with the same solution

Reply:
You are reading my mind 👍

Reply:
I see our orders slightly differ. I'll change what I have to what you have. I think your right that default should be last